Sources/SymbolKit/SymbolGraph/SymbolGraph.swift

```diff
@@ -62,13 +62,27 @@ public struct SymbolGraph: Codable {
 
     public static func _symbolToKeepInCaseOfPreciseIdentifierConflict(_ lhs: Symbol, _ rhs: Symbol) -> Symbol {
         if lhs.declarationContainsAsyncKeyword() {
-            return rhs
+            var result = rhs
+            result.addAlternateDeclaration(from: lhs)
+            return result
         } else if rhs.declarationContainsAsyncKeyword() {
-            return lhs
+            var result = lhs
+            result.addAlternateDeclaration(from: rhs)
+            return result
         } else {
             // It's not expected to ever end up here, but if we do, we return the symbol with the longer name
             // to have consistent results.
-            return lhs.names.title.count < rhs.names.title.count ? rhs : lhs
```

Comment:
If `title.count` is the same. Then the result of `_symbolToKeepInCaseOfPreciseIdentifierConflict(s1, s2)` will be different `_symbolToKeepInCaseOfPreciseIdentifierConflict(s2, s1)`.

But according to the comment, we expect a consistent results.

Should we use a more "stable" property here? eg. `names.title.hash`

Reply:
I wouldn't want to use a hash specifically, but we could check the lexicographical sorting of the identifiers if they're equal. That way it's still at least based on something visible.